packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -455,6 +455,23 @@ const Services = WebexPlugin.extend({
     return !!(catalog.findServiceUrlFromUrl(url));
   },
 
+  /**
+   * Converts the host portion of the url from default host
+   * to a priority host
+   * @param {string} url a service url that contains a default host
+   * @returns {string} a service url that contains the top priority host.
+   * @throws if url isn't a service url
+   */
+  convertUrlToPriorityHostUrl(url = '') {
+    const data = this.getServiceFromUrl(url);
+
+    if (!data) {
+      throw Error(`No service associated with url: [${url}]`);
+    }
+
+    return url.replace(data.defaultUrl, data.priorityHost);
+  },
+
```

Comment:
Great and efficient helper method. In this case, could you add a quick test to the PR to validate this method's functionality?